waku/v2/protocol/waku_filter/waku_filter.nim

```diff
@@ -187,44 +188,63 @@ method init*(wf: WakuFilter) =
 
   wf.handler = handle
   wf.codec = WakuFilterCodec
+  wf.timeout = filterTimeout
 
-proc init*(T: type WakuFilter, peerManager: PeerManager, rng: ref BrHmacDrbgContext, handler: MessagePushHandler): T =
+proc init*(T: type WakuFilter, peerManager: PeerManager, rng: ref BrHmacDrbgContext, handler: MessagePushHandler,timeout: Duration = WakuFilterTimeout): T =
   let rng = crypto.newRng()
   var wf = WakuFilter(rng: rng,
                       peerManager: peerManager, 
                       pushHandler: handler)

```

Comment:
If you add `timeout` to the `WakuFilter(...` object creation, you don't have to pass the argument to the second `init()` constructor :)

Reply:
resolved in bfc3104ca29a499cc4512975ba8e90e7f6071342